csrc/quantization/cutlass_w8a8/broadcast_load_epilogue_c2x.hpp

```diff
@@ -134,7 +136,31 @@ struct VisitorRowOrScalarBroadcast {
       auto coord_v = filter(tC_cRow);
       auto dst_v = filter(tC_rRow);
 
-      if (params_ptr->row_broadcast) {
+      // Fill dst_v with the scalar value
+      auto fill_dst = [&](Element const& value) {
```

Comment:
you created this `fill_dst` utility thats a copy of this `else` statement at line 171, why not use it there too?